libr/core/pseudo.c

```diff
@@ -1,7 +1,148 @@
 /* radare - LGPL - Copyright 2015-2016 - pancake */
 
 #include <r_core.h>
+#define TYPE_NONE		0
+#define TYPE_STR		1
+#define TYPE_SYM		2
+#define IS_CHAR(x)		(IS_UPPER(x) || IS_LOWER(x))
+#define IS_STRING(x)	(x+3<end && *x == 's' && *(x+1) == 't' && *(x+2) == 'r' && *(x+3) == '.')
```

Comment:
wrap all x with parenthesis to avoid unexpected  issues when using composed values